### DISCUSSION TITLE
How to add different sources to null-ls depending on project?

### DISCUSSION THREAD

**Discussion by ejazahm3d** [LazyVim/LazyVim#410] [Q&A]
Hi.
I have been trying to configure different sources for null-ls depending on project name. 
For example, in one project I want to use eslint for formatting but for another I require prettier. 
It would be great if someone can guide me or point me to some resources. 
Thanks :)

**Comment by folke** [comment]
I'm not sure how to do that. Best to check the docs on the null-ls project
